sympy/core/relational.py

```diff
@@ -1138,3 +1047,309 @@ def _eval_relation(cls, lhs, rhs):
+
+
+def is_ge(lhs, rhs):
+    """
+    Fuzzy bool for lhs is greater than or equal to rhs.
+
+    Returns True if lhs is greater than or equal to rhs
+    Returns False if lhs is not greater than rhs
+    Returns None if lhs cannot be compared to rhs, or if the comparison is indeterminate
+
+    is_le calls is_ge swapping rhs and lhs
+    is_lt calls is_ge and then fuzzy nots the result
+    is_gt calls is_le and fuzzy nots the result
+
+    so, to override inequality you just need to write an _eval_is_ge
+    using multiple dispatch
+
+    Examples
+    ========
+
+    >>> from sympy.core.relational import is_ge
+    >>> from sympy.core.sympify import _sympify
+    >>> is_ge(_sympify(2), _sympify(0))
+    True
+
+    >>> from sympy.core.relational import is_ge
+    >>> from sympy.core.sympify import _sympify
+    >>> is_ge(_sympify(0), _sympify(2))
+    False
+
+    >>> from sympy.abc import x
+    >>> from sympy.core.relational import is_ge
+    >>> from sympy.core.sympify import _sympify
+    >>> is_ge(_sympify(0), x)
+
+    >>> from sympy.core.relational import is_gt
+    >>> from sympy.core.sympify import _sympify
+    >>> is_gt(_sympify(2), _sympify(0))
+    True
+
+    >>> from sympy.core.relational import is_gt
+    >>> from sympy.core.sympify import _sympify
+    >>> is_gt(_sympify(0), _sympify(2))
+    False
+
+    >>> from sympy.core.relational import is_lt
+    >>> from sympy.core.sympify import _sympify
+    >>> is_lt(_sympify(0), _sympify(2))
+    True
+
+    >>> from sympy.core.relational import is_lt
+    >>> from sympy.core.sympify import _sympify
+    >>> is_lt(_sympify(2), _sympify(0))
+    False
+
+   """
+    if not (isinstance(lhs, Expr) and isinstance(rhs, Expr)):
+        raise TypeError("Can only compare inequalities with Expr")
+
+    retval = _eval_is_ge(lhs, rhs)
+
+    if retval is not None:
+        return retval
+    else:
+        n2 = _n2(lhs, rhs)
+        if n2 is not None:
+            # use float comparison for infinity.
+            # otherwise get stuck in infinite recursion
+            if n2 in (S.Infinity, S.NegativeInfinity):
+                n2 = float(n2)
+            return _sympify(n2 >= 0)
+        if lhs.is_extended_real and rhs.is_extended_real:
+            if (lhs.is_infinite and lhs.is_extended_positive) \
+                or (rhs.is_infinite and rhs.is_extended_negative):
+                return True
+            diff = lhs - rhs
+            if diff is not S.NaN:
+                rv = diff.is_extended_nonnegative
+                if rv is not None:
+                    return rv
+
+
+def is_neq(lhs, rhs):
+    """Fuzzy bool for lhs does not equal rhs.
+
+    See the docstring for is_eq for more
+    """
+    return fuzzy_not(is_eq(lhs, rhs))
+
+
+def is_eq(lhs, rhs):
+    """
+    Fuzzy boolean used to check if lhs is equal, mathematically
+    to rhs.
+    Returns True or False if the two are comparable, or None if the two are incomparable
+    or if the comparison is indeterminate.
```

Comment:
The word comparable is tricky because sympy uses it to mean something different:
```julia
In [1]: (1 + sqrt(2)).is_comparable                                                                                                            
Out[1]: True
```
That meaning of comparable is "the expression is real-valued and can be evaluated to a simple `Float`". The sense of comparability is that that makes it usually possible to evaluate an inequality `a<b` by numerical comparison.

How about something like:

Fuzzy bool representing mathematical equality between lhs and rhs.

If lhs and rhs are easily shown to be equal (or unequal) then True (or False) is returned. Otherwise None is returned meaning that lhs and rhs may or may not be equal.

Notes:

This function is intended to give a relatively fast determination and deliberately does not attempt slow calculations that might help in obtaining a determination of True or False in more difficult cases. The evaluation of an `Equality` uses `is_eq` internally so if `is_eq(a, b)` gives True (or False) then `Eq(a, b)` will give `S.true` or `S.false` and vice versa. When `is_eq(a, b)` returns `None`, `Eq(a, b)` will give an unevaluated `Eq`.

Since `is_eq` is primarily an _internal_ function it is expected that the arguments will be sympified already.